### DISCUSSION TITLE
Publish to crates.io

### DISCUSSION THREAD

**Discussion by augustocdias** [neovide/neovide#956] [Ideas]
I see there's some work on publishing it to packages repositories... Could you publish the binary to crates.io in the meantime?

It is much more comfortable to just run `cargo install neovide` than to clone, pull and build manually every time it is updated.

**Comment by Kethku** [comment]
This task is tracked here https://github.com/neovide/neovide/issues/839 maybe I don't understand things properly but my I thought to publish to crates.io you have to have all the dependencies also on crates io. I was pretty sure you can't just publish a binary. Maybe I'm wrong?

**Reply by augustocdias** [reply]
You publish the code. Does it depend on non open source code? If it depends on native libraries is fine. It is also fine to depend on GitHub libraries instead of crates.io. It will only be more susceptible to errors if the code on GitHub changes.

**Reply by MultisampledNight** [reply]
For the time being, it's possible to use `cargo install --git https://github.com/neovide/neovide` as well.
